client/src/pages/admin/mentor-registry.tsx

```diff
@@ -46,17 +60,21 @@ function Page(props: Props) {
   const loadData = useCallback(
```

Comment:
I'd propose to add proper deps array or remove `useCallback` here

Reply:
fixed